src/seed_and_multiplier_generators.rs

```diff
@@ -8,11 +8,45 @@ pub fn time_num_generator<F: FeltTrait>() -> F {
     let now = chrono::Local::now();
     let midnight = chrono::NaiveTime::from_hms(0, 0, 0);
     let time_since_midnight = now.time() - midnight;
+    let mut felt = F::new(time_since_midnight.num_milliseconds() as u64);
 
-    F::new(time_since_midnight.num_milliseconds() as u64)
+    while felt == F::zero() {
+        let now = chrono::Local::now();
+        let time_since_midnight = now.time() - midnight;
+        felt = F::new(time_since_midnight.num_milliseconds() as u64);
+    }
+
+    felt
 }
```

Comment:
a while is to much ... the only time when we can create something like zero for timegeneration is at 0:00:00 for that we just need to do an if statement, because we the time that goes by while we compare is enough to create a non zero number